File: algorithm/src/main/java/baekjoon/p10174/Main.java
package baekjoon.p10174;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();

        while (n-- > 0) {
            System.out.println(solve(sc.nextLine()));
        }
    }

    private static String solve(String input) {
        String lower = input.toLowerCase();
        String rev = new StringBuilder(lower).reverse().toString();

        if (lower.equals(rev)) {
            return "Yes";
        } else {
            return "No";
        }
    }

}

File: algorithm/src/main/java/baekjoon/p10174/Main.kt
package baekjoon.p10174

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        println(solve(readLine()!!))
    }
}

private fun solve(input: String): String {
    val lower = input.toLowerCase()
    val rev = lower.reversed()

    return when (lower == rev) {
        true -> "Yes"
        false -> "No"
    }
}